xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1398,6 +1410,23 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
                     gf_upcall_list[i], fop_hits, "0", "0", "0");
     }
 
+    /* To measure FOP_HITS_PER_SEC consider time difference
+       between last profile stat started and last fop hit.
+       In case if last_fop_hit value is 0 then consider now
+       to measure the duration
+    */
+    if (conf->last_fop_hit) {
```

Comment:
Where is `last_fop_hit` reset to 0 ? If we receive a fop in an interval, but none in the next interval, `last_fop_hit` won't be 0, but it will smaller than `stats->started_at`. Or I missed something ?

Reply:
I have not reset last_fop_hit to 0. To avoid this kind of scenario I have put a condition we will last_fop_hits timestamp only while it is greater than the last stat started.